sds/src/rule.rs

```diff
@@ -4,6 +4,7 @@ use crate::match_action::MatchAction;
 use crate::path::Path;
 use serde_with::{serde_as, DefaultOnNull};
 
+#[serde_as]
```

Comment:
@trosenblatt 
>What are those labels used for exactly ?

`Labels`  at the rule level are like [other labels at the scanner level](https://github.com/DataDog/dd-sensitive-data-scanner/blob/main/sds/src/scanner/mod.rs#L44) to hold `key:value` for observability. They mostly refer to the term "label" defined here
https://github.com/DataDog/dd-sensitive-data-scanner/blob/main/sds/src/observability/labels.rs#L3

I decided to not expose directly `Labels`/`Label` in the RuleConfig and use instead Vec<(String, String)> so the dependency on `metrics` is fully internal to the package.

Reply:
But what are they used for ?

Reply:
they are used in metrics to tag metric points by org/product

Reply:
Ok thks I think Labels is confusing in that case why not calling it MetricLabels ?

Reply:
About renaming it `Labels` to `MetricLabels`, I'm fine doing it but at the same time I think it's fine as it is and leave a more generic name that can be re-used for trace/log labels. I can rename these in another PR if needed